lookout/style/typos/utils.py

```diff
@@ -0,0 +1,177 @@
+from itertools import chain
+
+from gensim.models.fasttext import FastText
+import numpy
+import pandas
+
+
+def collect_embeddings(fasttext: FastText, tokens) -> numpy.ndarray:
+    return numpy.array([fasttext.wv[token] for token in tokens])
+
+
+def read_frequencies(file: str) -> dict:
+    """
+    Read token frequencies from the file.
+    :param file: Path to the file containing tokens with frequencies.
+                Every line has format "token count"
+    :return: Dictionary of tokens frequencies
+    """
+    frequencies = {}
+    with open(file, "r") as f:
+        for line in f:
+            split = line.split()
+            frequencies[split[0]] = int(split[1])
+    return frequencies
+
+
+def read_vocabulary(file: str) -> list:
+    """
+    Read vocabulary tokens from the text file
+    :param file: Text file used to generate vocabulary of corrections candidates.
+            First token in every line split is added to the vocabulary
```

Comment:
Please align comments at the new line as in all other code - example https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/features.py#L222